pkg/cloudprovider/aws/amifamily/ami.go

```diff
@@ -39,10 +39,10 @@ func (p *AMIProvider) Get(ctx context.Context, instanceType cloudprovider.Instan
 	}
 	output, err := p.ssm.GetParameterWithContext(ctx, &ssm.GetParameterInput{Name: aws.String(ssmQuery)})
 	if err != nil {
-		return "", fmt.Errorf("getting ssm parameter, %w", err)
+		return "", fmt.Errorf("getting ssm parameter %q, %w", ssmQuery, err)
 	}
 	ami := aws.StringValue(output.Parameter.Value)
 	p.cache.SetDefault(ssmQuery, ami)
-	logging.FromContext(ctx).Debugf("Discovered %s for query %s", ami, ssmQuery)
```

Comment:
It just causes it to be quoted, since it's input we generate that gets sent to an API call, it makes it easier to see if some  weird character happens to make it in somehow.  Not much difference, but there's a tiny chance it will save some debugging at some point.

Reply:
Got it. I'm happy with that.